example/test.js

```diff
@@ -0,0 +1,9992 @@
+goog.exportSymbol('proto.jspb.test.HasExtensions', null, global);
+goog.exportSymbol('proto.jspb.test.IndirectExtension', null, global);
+goog.exportSymbol('proto.jspb.test.Int64Types', null, global);
+goog.exportSymbol('proto.jspb.test.IsExtension', null, global);
+goog.exportSymbol('proto.jspb.test.MapValueEnumNoBinary', null, global);
+goog.exportSymbol('proto.jspb.test.MapValueMessageNoBinary', null, global);
+goog.exportSymbol('proto.jspb.test.MineField', null, global);
+goog.exportSymbol('proto.jspb.test.OptionalFields', null, global);
+goog.exportSymbol('proto.jspb.test.OptionalFields.Nested', null, global);
+goog.exportSymbol('proto.jspb.test.OuterEnum', null, global);
+goog.exportSymbol('proto.jspb.test.OuterMessage', null, global);
+goog.exportSymbol('proto.jspb.test.OuterMessage.Complex', null, global);
+goog.exportSymbol('proto.jspb.test.Simple1', null, global);
+goog.exportSymbol('proto.jspb.test.Simple2', null, global);
+goog.exportSymbol('proto.jspb.test.SpecialCases', null, global);
+goog.exportSymbol('proto.jspb.test.TestAllowAliasEnum', null, global);
+goog.exportSymbol('proto.jspb.test.TestClone', null, global);
+goog.exportSymbol('proto.jspb.test.TestCloneExtension', null, global);
+goog.exportSymbol('proto.jspb.test.TestEndsWithBytes', null, global);
+goog.exportSymbol('proto.jspb.test.TestGroup', null, global);
+goog.exportSymbol('proto.jspb.test.TestGroup.OptionalGroup', null, global);
+goog.exportSymbol('proto.jspb.test.TestGroup.RepeatedGroup', null, global);
+goog.exportSymbol('proto.jspb.test.TestGroup.RequiredGroup', null, global);
+goog.exportSymbol('proto.jspb.test.TestGroup1', null, global);
+goog.exportSymbol('proto.jspb.test.TestLastFieldBeforePivot', null, global);
+goog.exportSymbol('proto.jspb.test.TestMapFieldsNoBinary', null, global);
+goog.exportSymbol('proto.jspb.test.TestMessageWithOneof', null, global);
+goog.exportSymbol('proto.jspb.test.TestMessageWithOneof.DefaultOneofACase', null, global);
+goog.exportSymbol('proto.jspb.test.TestMessageWithOneof.DefaultOneofBCase', null, global);
+goog.exportSymbol('proto.jspb.test.TestMessageWithOneof.PartialOneofCase', null, global);
+goog.exportSymbol('proto.jspb.test.TestMessageWithOneof.RecursiveOneofCase', null, global);
+goog.exportSymbol('proto.jspb.test.TestReservedNames', null, global);
+goog.exportSymbol('proto.jspb.test.TestReservedNamesExtension', null, global);
+goog.exportSymbol('proto.jspb.test.extendTestLastFieldBeforePivotField', null, global);
+goog.exportSymbol('proto.jspb.test.simple1', null, global);
+/**
+ * Generated by JsPbCodeGenerator.
+ * @param {Array=} opt_data Optional initial data array, typically from a
+ * server response, or constructed directly in Javascript. The array is used
+ * in place and becomes part of the constructed object. It is not cloned.
+ * If no data is provided, the constructed object will be empty, but still
+ * valid.
+ * @extends {jspb.Message}
+ * @constructor
+ */
+proto.jspb.test.Empty = function(opt_data) {
```

Comment:
Shouldn't this be `export function Empty(opt_data) {`?

Reply:
Fixed.